homeassistant/components/light/tellduslive.py

```diff
@@ -0,0 +1,65 @@
+"""
+Support for Tellstick switches using Tellstick Net.
+
+This platform uses the Telldus Live online service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.tellduslive/
+
+"""
+import logging
+
```

Comment:
It was actually removed a while ago because of a bug: https://github.com/home-assistant/home-assistant/issues/913#issuecomment-172497631 (https://github.com/molobrakos/home-assistant/commit/d43101f22a045b58b37cbdaedae19185d9614b53)
Has something maybe changed since then?